### PR TITLE
Add blog post and new author

### PR DESCRIPTION
**Pull request by juliaklindpaintner**
# Pull request summary

- Adds @amandarkennedy as an author
- Adds blog post "Customer experience: beyond surveys"

### CODE REVIEW

**Review by jskinne3 — CHANGES_REQUESTED** [review]
Suggest removing a stray `-` character

I'm trying to figure out why the preview build isn't working

File: _posts/2024-03-05-customer-experience-beyond-surveys.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+date: 2024-03-05
+title: >
+ Customer experience: beyond surveys
+ authors: 
+  - amanda-kennedy
+tags: 
+- 18f
+- how we work
+-
```
[the remark this answers is not in the document]
Comment (juliaklindpaintner) [reply _posts/2024-03-05-customer-experience-beyond-surveys.md]: Ah thank you for catching! I meant to add a third tag. Added now!

**Review by jskinne3 — APPROVED** [review]
Great!

**Comment by juliaklindpaintner** [comment]
@amandarkennedy Can you take a look at [this preview of the post](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/cx-beyond-surveys-amanda-kennedy/2024/03/05/customer-experience-beyond-surveys/) and see if you're happy with it? @quarterback is OOO today, but I will merge to publish — just wanted to get another set of eyes on it first.

**Comment by amandarkennedy** [comment]
@juliaklindpaintner the only thing I noticed is that the following methods-related headers are now h1 when they were just bolded normal text in the draft. At the very least, I would prefer they be h2, but otherwise looks good. 
Screenshot 2024-03-05 at 9 24 25 AM

**Comment by juliaklindpaintner** [comment]
@amandarkennedy They are actually h2s already (the other headings are h1), but let me adjust to make them just be bolded.

**Comment by amandarkennedy** [comment]
@juliaklindpaintner that would be great, thanks!

**Comment by juliaklindpaintner** [comment]
@amandarkennedy How about [this](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/cx-beyond-surveys-amanda-kennedy/2024/03/05/customer-experience-beyond-surveys/)?

**Comment by amandarkennedy** [comment]
@juliaklindpaintner this looks great! One more thing I just realized: "Choose your method" doesn't follow the same framing of the other headers in the form of a question. Would it be possible to change it to "What method should you choose?" I am happy to do that (or leave as is).

**Comment by juliaklindpaintner** [comment]
I see what you're saying @amandarkennedy. The last heading is also a different format ("Implement") — are you okay with that one?

**Comment by amandarkennedy** [comment]
@juliaklindpaintner good question, and yes! I am okay with keeping `Implement` as is. Thanks for checking!

**Comment by juliaklindpaintner** [comment]
Okay! One last check of [this preview](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/cx-beyond-surveys-amanda-kennedy/2024/03/05/customer-experience-beyond-surveys/)? If it looks good, you can go ahead and approve this PR :)

**Comment by quarterback** [comment]
I'm around. and it looks good

**Pull request merged**